specification/impact/Impact.Management/workloadImpacts.tsp

```diff
@@ -59,9 +59,10 @@ model WorkloadImpactProperties is ResourceProperties {
   @doc("Details about connectivity issue. Applicable when root resource causing the issue is not identified. For example, when a VM is impacted due to a network issue, the impacted resource is identified as the VM, but the root cause is the network. In such cases, the connectivity field will have the details about the network issue")
   connectivity?: Connectivity;
 
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "Existing use case"
   #suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" "Existing use case"
   @doc("Additional fields related to impact, applicable fields per resource type are list under /impactCategories API")
-  additionalProperties?: {};
+  additionalProperties?: Record<unknown>;
```

Comment:
The property name `additionalProperties` combined with `Record<unknown>` is a red flag for me.
Do you want the request body payload be
```json
{
  "additionalProperties": {
    "prop1": "value1",
    "prop2": {}
  }
}
```
Or 

```json
{
  "prop1": "value1",
  "prop2": "value2"
}
```
?

Reply:
In their examples, they seem using the former
https://github.com/Azure/azure-rest-api-specs/blob/3e7c8399d85aeb8e4504214fe38ccce623ccbe4c/specification/impact/Impact.Management/examples/2024-05-01-preview/WorkloadImpacts_ListBySubscription.json#L24-L27

However I am not sure whether the example is correct...

Reply:
Yes this looks not so common, but I have confirmed with the service team, this is expected.
Usually we do not design the payload in this way, but now that we have it in this way, the spec is correctly showing how it looks like in the payload.